index.html

```diff
@@ -5,31 +5,71 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <link rel="stylesheet" href="./css/style.css">
+    <link rel="stylesheet" href="./style.css">
     <title>Awesome Books</title>
 </head>
 
 <body>
+    <header>
```

Comment:
- [ ] It would be better if we fix this bug, once the app is started all the 3 pages appears on the webpage as we can see in the screenshot below
![Screenshot from 2023-03-23 10-07-30](https://user-images.githubusercontent.com/46388113/227172601-4db708e1-29f4-4ccf-afba-2ee757d4cd3b.png)